m46x/StdDriver/drv_emac/synopGMAC_network_interface.c

```diff
@@ -0,0 +1,942 @@
+/* ===================================================================================
+ * Copyright (c) <2009> Synopsys, Inc.
+ *
```

Comment:
Please mention Apache-2.0 license as below in the each file  header comments

SPDX-License-Identifier: Apache-2.0

Reply:
Fixed in commit https://github.com/zephyrproject-rtos/hal_nuvoton/commit/17e131d6cf28744a149250e72113611fbe278370